Cards/CardContainer.gd

```diff
@@ -240,6 +244,7 @@ func _bind_card_input(card: CardWorld) -> void:
 	card_click_handler.on_click.connect(_on_card_clicked.bind(card))
 	card_click_handler.on_mouse_hovering.connect(_on_card_hovering.bind(card))
 	card_click_handler.on_unhover.connect(_on_card_unhovered.bind(card))
+	card_click_handler.on_right_click.connect(on_right_click.bind(card))
```

Comment:
The problem with having the right click in the ClickHandler is that you always have to right click the card to cancel it. This is a bit annoying when you have a SINGLE_TARGET type card, because it's not going to be at your mouse position

Reply:
So maybe we can check for the right click in `_process` using `Input.is_mouse_button_pressed` or something similar. This way it isn't checking for a click on a specific card

Reply:
Ye, that's fair, I'll do that